pkg/envoy/xds_server.go

```diff
@@ -1610,7 +1611,7 @@ func getWildcardNetworkPolicyRule(version *versioned.VersionHandle, selectors po
 			// Issue a warning if this port-0 rule is a redirect.
 			// Deny rules don't support L7 therefore for the deny case
 			// l7.IsRedirect() will always return false.
-			log.Warningf("L3-only rule for selector %v surprisingly requires proxy redirection (%v)!", sel, *l7)
+			s.logger.Warn("L3-only rule for selector surprisingly requires proxy redirection!", logfields.Selector, sel, "l7", *l7)
```

Comment:
missing loggingfields

Reply:
this function is not migrated to slog as it's used by policy.

It's part of part 2 (see PR desc)

Reply:
We could probably remove the logging of `*l7` from here, I dont recall ever seeing this being logged, so there is no point having that much detail here.

Reply:
The linter will fail because of this `"l7"` not being a "logfields."

Reply:
ok, removed logging logfields `l7` (`*l7`) completely